core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -958,6 +959,16 @@ public SqlNode toSql(RexWindowBound rexWindowBound) {
       }
     }
 
+    public SqlLiteral toSql(RexWindowExclusion exclude) {
+      switch (exclude) {
+      case EXCLUDE_CURRENT_ROW: return SqlWindow.createExcludeCurrentRow(POS);
+      case EXCLUDE_GROUP: return SqlWindow.createExcludeGroup(POS);
+      case EXCLUDE_TIES: return SqlWindow.createExcludeTies(POS);
+      }
+      assert exclude == RexWindowExclusion.EXCLUDE_NO_OTHER;
```

Comment:
Nit: why not a regular case and throwing on `default`? 

I am also wondering if we want to omit printing the exclusion when there is no exclusion, it would keep the generated SQL backward compatible to what we have been generating now. WDYT?

Reply:
I've replace this assert with exception. 
Regarding generated SQL, we do ignore the exclude in case of `Exclude no others` in `SqlWindow.unparse`.